app/assets/javascripts/angular/controllers/dashboard.js

```diff
@@ -12,30 +12,14 @@ dashboardController.controller('DashboardCtrl', ['$rootScope', '$scope', '$http'
       success(function(data, status, headers, config) {
         var result = data['result'];
         $scope.terror = result['global_terror']['total'];
-        $scope.activity = result['global_terror']['activity'];
         $scope.paused = result['timer']['paused'];
         $scope.countries = result['countries'];
         $scope.controlMessage = result['timer']['control_message'];
         $scope.round = result['timer']['round'];
-        $scope.rioters = result['global_terror']['rioters'];
-        if (result['alien_comms'] == true) {
-          $("body").addClass("alien");
-          $("p").addClass("alien");
-          $("td").addClass("alien");
-        } else {
-          $("body").removeClass("alien");
-          $("p").removeClass("alien");
-          $("td").removeClass("alien");
-        }
-        if (result['vatican_alien_comms'] == true) {
-          console.log("Adding class");
-          console.log($(".Vatican"));
-          $('.Vatican').addClass('alien');
 
-        } else {
-          console.log("Removing class");
-          $(".Vatican").removeClass("alien");
-        }
+        $('.body').toggleClass('alien', result['alien_comms']);
+        $('.Vatican').toggleClass('alien', result['vatican_alien_comms']);
```

Comment:
I didn't remove it since I wasn't sure if it was coming later, but fyi there is currently no class `".Vatican"` used anywhere in the app except for right here. This will never trigger.